### PR TITLE
chore: move network models from base [WPB-8829]

### PR DESCRIPTION
**Pull request by Garzas**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-8829" title="WPB-8829" target="_blank">TaskWPB-8829</a>  [Android] Configure the project to run UI Compose (espresso) tests
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?
Moved network models from base to api module according to https://github.com/wireapp/kalium/pull/2822#pullrequestreview-2129684843

### CODE REVIEW

File: logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt [hunk 1]
```diff
@@ -415,23 +415,23 @@ internal class ConversationMapperImpl(
 
     private fun ConversationResponse.getProtocolInfo(mlsGroupState: GroupState?): ProtocolInfo {
         return when (protocol) {
-            ConvProtocol.MLS -> ProtocolInfo.MLS(
+            com.wire.kalium.network.api.authenticated.conversation.ConvProtocol.MLS -> ProtocolInfo.MLS(
```
Comment (vitorhugods) [inline logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt:418]: 🤔 
Could be imported

File: logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt [hunk 2]
```diff
@@ -415,23 +415,23 @@ internal class ConversationMapperImpl(
...
                 groupId ?: "",
                 mlsGroupState ?: GroupState.PENDING_JOIN,
                 epoch ?: 0UL,
                 keyingMaterialLastUpdate = DateTimeUtil.currentInstant(),
                 ConversationEntity.CipherSuite.fromTag(mlsCipherSuiteTag)
             )
 
-            ConvProtocol.MIXED -> ProtocolInfo.Mixed(
+            com.wire.kalium.network.api.authenticated.conversation.ConvProtocol.MIXED -> ProtocolInfo.Mixed(
```
Comment (vitorhugods) [inline logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt:426]: Same

File: logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt [hunk 3]
```diff
@@ -415,23 +415,23 @@ internal class ConversationMapperImpl(
...
                 groupId ?: "",
                 mlsGroupState ?: GroupState.PENDING_JOIN,
                 epoch ?: 0UL,
                 keyingMaterialLastUpdate = DateTimeUtil.currentInstant(),
                 ConversationEntity.CipherSuite.fromTag(mlsCipherSuiteTag)
             )
 
-            ConvProtocol.PROTEUS -> ProtocolInfo.Proteus
+            com.wire.kalium.network.api.authenticated.conversation.ConvProtocol.PROTEUS -> ProtocolInfo.Proteus
```
Comment (vitorhugods) [inline logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt:434]: Same

**Review by vitorhugods — COMMENTED** [review]
LGTM

All files change just packages or imports, so the compiler should do the rest.

Just one file has a few missing imports, maybe.

File: network/src/commonMain/kotlin/com/wire/kalium/network/api/v0/unauthenticated/LoginApiV0.kt [hunk 4]
```diff
@@ -80,7 +79,7 @@ internal open class LoginApiV0 internal constructor(
         }
     }.flatMap { accessTokenDTOResponse ->
         with(accessTokenDTOResponse) {
-            cookies[RefreshTokenProperties.COOKIE_NAME]?.let { refreshToken ->
+            cookies[com.wire.kalium.network.api.model.RefreshTokenProperties.COOKIE_NAME]?.let { refreshToken ->
```
Comment (saleniuk) [inline network/src/commonMain/kotlin/com/wire/kalium/network/api/v0/unauthenticated/LoginApiV0.kt:82]: here as well 😅

**Review by saleniuk — APPROVED** [review]
I just found one more place with missing import, but approve already as it's easy to change

**Review by mchenani — APPROVED** [review]
197ffed1d81f87b5d7a0d21e27c789d7

**Pull request merged**